frontend/pages/shelter/pet/[id].tsx

```diff
@@ -1,36 +1,57 @@
+import { Animal } from "@contract";
 import { Breadcrumb } from "antd";
 import { Content } from "antd/lib/layout/layout";
-import { PetData, Species, Status } from "common/enums";
 import PetDetailsSection from "components/shelter/pet/PetDetailsSection";
 import ProspectiveAdopters from "components/shelter/pet/ProspectiveAdopters";
 import ShelterLayout from "layouts/shelter/ShelterLayout";
+import { MenuKey } from "layouts/shelter/ShelterLayout/LeftMenu";
 import { useRouter } from "next/dist/client/router";
 import React, { useEffect, useState } from "react";
 import styled from "styled-components";
 
 export default function PetDetails() {
 	const router = useRouter();
 	const { id } = router.query;
```

Comment:
For convenience, if other places want to use `id`, it will be typed.
```suggestion
	const id = router.query.id as string;
```